src/Repositories/RedisSupervisorRepository.php

```diff
@@ -71,7 +71,7 @@ public function get(array $names)
     {
         $records = $this->connection()->pipeline(function ($pipe) use ($names) {
             foreach ($names as $name) {
-                $pipe->hmget('supervisor:'.$name, ['name', 'master', 'pid', 'status', 'processes', 'options']);
+                $pipe->hmget('supervisor:'.$name, ['name', 'master', 'pid', 'cpu', 'mem', 'status', 'processes', 'options']);
```

Comment:
Hmm, I hadn't considered the sequence of the parameters here. It's probably best if you add them at the end if people rely on the sequence to determine the values (see the array keys below). If they're new values at the end of the array it's not breaking.

Reply:
Fixed